templates/foreman_supervisory_authority.yaml.erb

```
@@ -0,0 +1,12 @@
+:foreman_supervisory_authoriy:
```

Is there a typo here?
```suggestion
:foreman_supervisory_authority:
```

---

Thank's for noticing this! This is actually a typo in the plugin itself (from which the settings file was copied). See: https://github.com/timogoebel/foreman_supervisory_authority/pull/6

---

I fixed the template in this MR. I guess we should wait for the new plugin version to become available so we don't brake Foreman